Assets/MixedRealityToolkit.Extensions/Sharing/SpatialAlignment/Common/SpatialCoordinateServiceBase.cs

```diff
@@ -156,9 +157,11 @@ public async Task<bool> TryDiscoverCoordinatesAsync(CancellationToken cancellati
 
                 return true;
             }
-            finally
+            catch (Exception e)
             {
```

Comment:
We shouldn't swallow exceptions here, that is a very bad practice. We can log, and then rethrow by doing "throw;"

Reply:
sounds good.